doc/distrib/Samples/en-US/Core/Core_Math.dyn

```diff
@@ -10720,8 +10720,8 @@
         "PinnedNode": "71cb8f97deb648bb8e7824547b43f3aa"
       }
     ],
-    "X": -71.98682450110687,
-    "Y": 2008.3459017897544,
-    "Zoom": 0.28542192141810496
+    "X": 51.34875002226386,
+    "Y": 800.7886260085731,
+    "Zoom": 0.19068736946261172
```

Comment:
I wonder if it's the change of view port making the smoke tests fail, @EdwardChaplinArchilizer maybe revert the changes here. FYI: @reddyashish @zeusongit

Reply:
From the discussion on slack, I think @sm6srw and @avidit are looking into it.